pkg/arvo/app/dojo.hoon

```diff
@@ -883,10 +882,26 @@
       ::
       =/  sam=vase  :(slop ven poz kev)
       ?.  (~(nest ut p.som) | p.sam)
+        =/  gav=vase  (slot 12 gat)
+        ?.  (~(nest ut p.ven) | p.gav)
+          ::  event arguments
+          ::
+          ~>  %slog.1^leaf+"dojo: invalid generator event sample"
+          ~>  %slog.1^leaf+"dojo: nest-need"
+          ~>  %slog.0^(skol p.ven)
+          ~>  %slog.1^leaf+"dojo: nest-have"
+          ~>  %slog.0^(skol p.gav)
+          !!
```

Comment:
These were already like this, but I think it'd be better to put the errors in the trace instead of eagerly slogging them. You can do that with `~_`

Reply:
Like so?